bin/release_chart

```diff
@@ -107,8 +107,28 @@ else #if installing from local, "unpacked" chart, then restore dependencies firs
   helm dependency build $chart
 fi
 
-msg "Installing chart"
-helmCmd="helm upgrade $release $chart --install $repoUrl $version $customFlags --force --reset-values $namespace $customVals"
-
-msg "$helmCmd"
-eval "$helmCmd"
+if [ $ACTION == 'push' ]
+then
+	msg "Packaging the chart..."
+	helm package $chart $version --destination /dev/shm/ 1>/dev/null
+	packedChartName=$(ls /dev/shm/)
```

Comment:
I think it will be better to get the package name from the command output, like in this example: https://github.com/Codefresh-Examples/Examples/blob/master/helm-package-museum/codefresh.yaml